README.md

```diff
@@ -6,9 +6,12 @@ We plan and keep track of our efforts on the [Community Project Board](https://g
 
 ## Bootstrapping
 
-This repository is the home to efforts to bring a formal _community architecture_ process to the Operate First initiative.
-The scope of this initiative is large enough with a multi-community impact to warrant going from an organic community effort to a managed community.
-This initiative is in the beginning stages (bootstrapping) of defining and performing this work.
+This repository (https://github.com/operate-first/community) is the home to
+efforts to bring a formal _community architecture_ process to the Operate First
+initiative. The scope of this initiative is large enough with a multi-community
+impact to warrant going from an organic community effort to a managed
+community. This initiative is in the beginning stages (bootstrapping) of
+defining and performing this work.
```

Comment:
Can you return the single sentence per line for this section? Or we can go back and fix it later, but that is messier.

Reply:
Sure, but you explain why? Single-sentence-per-line renders as a regular paragraph (https://github.com/operate-first/community#bootstrapping). Is this meant to be a bullet list?

Reply:
Sorry, missed the question back.

In MarkDown (and AsciiDoc) you can have a single line per sentence for every sentence in a paragraph. The empty line is what denotes the paragraph break.

This way it is easier to review written content. In particular, copyediting work is much easier when you may have a series of small edits in one paragraph, or one edit in the middle of a long paragraph.

This also allows for a comment stream to happen for each suggested change or review question, just within that one sentence. If the next sentence has a series of comments, they are separated because they are on different line numbers.

This guidance is part of the overall docs style guide that we have partially written and need to publish soonest.